tests/order_modifiers_integration/Coupons/Create_Coupon_Modifiers_Test.php

```diff
@@ -31,6 +37,58 @@ public function does_table_render_correctly() {
 		$this->assertMatchesHtmlSnapshot( $this->get_table_display() );
 	}
 
+	/**
+	 * @test
+	 */
+	public function does_table_render_correctly_with_uses() {
+		for ( $i = 0; $i < 20; $i++ ) {
+			$insert_data = [
+				'modifier'                    => 'coupon',
+				'order_modifier_amount'       => (float) $i,
+				'order_modifier_sub_type'     => $i % 2 ? 'percent' : 'flat',
+				'order_modifier_slug'         => sprintf( 'test_coupon_%02d', $i ),
+				'order_modifier_display_name' => "Test Coupon Insert {$i}",
+
+			];
+
+			// Insert the modifier and store the ID.
+			$modifier = $this->upsert_order_modifier_for_test( $insert_data );
+
+			// Every other one, add a usage limit.
+			if ( $i % 2 ) {
+				$meta_repo = tribe( Order_Modifiers_Meta::class );
+				$meta_repo->upsert_meta(
+					new Order_Modifier_Meta(
+						[
+							'order_modifier_id' => $modifier->id,
+							'meta_key'          => 'usage_limit',
+							'meta_value'        => 20,
+						]
+					)
+				);
+			}
+
+			// For the first 10 coupons, use them.
+			if ( $i < 10 ) {
+				$this->add_coupon_use( $modifier->id, $i + 1 );
+			}
+		}
+
+		// Clear the request data to avoid conflicts with other tests.
+		unset( $_REQUEST['id'], $_REQUEST['edit'], $_POST['modifier_id'], $_POST['edit'] );
```

Comment:
This should be done in an `@after` or `@before` or both method.

Reply:
@dpanta94 This was copied from your code 😅

Reply:
Updated